Q3.js

```diff
@@ -0,0 +1,15 @@
+//In the following shopping cart add, remove, edit items
+
+const shoppingCart = ['Milk', 'Coffee', 'Tea', 'Honey']
+// add 'Meat' in the beginning of your shopping cart if it has not been already added
+// add Sugar at the end of you shopping cart if it has not been already added
+// remove 'Honey' if you are allergic to honey
+// modify Tea to 'Green Tea'
+shoppingCart.unshift('meat');
+console.log(shoppingCart);
+shoppingCart.push('sugger');
```

Comment:
@Fakhara77 Quesion `add Sugar at the end of you shopping cart if it has not been already added` add only if it is not already added. But you are blindly adding it.

Reply:
correct my spelling.